app/workers/send_new_cycle_has_started_email_to_candidates_worker.rb

```diff
@@ -17,4 +17,27 @@ def perform
       )
     end
   end
+
+  def relation
```

Comment:
How do you feel about running this query in a Production console to get an idea of the number of Candidates that will receive this? I don't know if it's worth comparing it to the `GetUnsuccessfulAndUnsubmittedCandidates` numbers 🤷‍♂️

Reply:
Will do.

Reply:
As of today: 
92,906 candidates before this change
84,902 candidates after this change
8004 fewer emails sent on the day. 

I would expect some of those 84902 to create 2025 applications between now and when we send it on Thursday.